core/src/main/java/uk/gov/justice/services/core/enveloper/Enveloper.java

```diff
@@ -37,15 +40,15 @@
 @ApplicationScoped
 public class Enveloper {
 
-    private final ObjectToJsonValueConverter objectToJsonValueConverter;
+    Clock clock;
 
-    //for CDI proxying only
-    protected Enveloper() {
-        this.objectToJsonValueConverter = null;
-    }
+    ObjectToJsonValueConverter objectToJsonValueConverter;
+
+    public Enveloper() {}
```

Comment:
Caused by: javax.enterprise.inject.UnproxyableResolutionException: WebBeans with api type with normal scope must be proxyable.
uk.gov.justice.services.core.enveloper.Enveloper has no explicit no-arg constructor!A public or protected constructor without args is required!